src/index.ts

```diff
@@ -0,0 +1,58 @@
+import "dotenv/config";
+import App from "./app";
+import database from "./database/database";
+import DateHandlerService, {
+  IDateHandlerService,
+} from "./date-handler/date-handler.service";
+import EmployeeController from "./employee/employee.controller";
+import EmployeeService, { IEmployeeService } from "./employee/employee.service";
+import InvoicesController from "./invoices/invoices.controller";
+import InvoicesFacade, { IInvoiceFacade } from "./invoices/invoices.facade";
+import InvoiceService, { IInvoicesService } from "./invoices/invoices.service";
+import SenderService, { ISenderService } from "./sender/sender.service";
+import FilesUploaderController from "./uploader/uploader.controller";
+import FilesUploaderService, {
+  IFilesUploaderService,
+} from "./uploader/uploader.service";
+
+const port = Number(process.env.PORT);
```

Comment:
Dobrze by było jeśli wyciągnąłbyś zmienne typu process env do czegoś co je waliduje.

Pozwala to w sensowny sposób napisać jakiej zmiennej ci brakuje/dodać defaultowe zamiast męczyć się z wiadomosciami systemowymi.

Reply:
ok zrobione